.github/workflows/check.yml

```diff
@@ -188,7 +188,7 @@ jobs:
         with:
           name: terminusdb-server-docker-image
 
-      - run: make download-lint
+      - run: make download-lint && chmod -R 777 .deps
```

Comment:
Because the user does not exist locally in the CI but only in the Docker. Or I'd have to create the same user in CI as well, which is of course possible but is a bit overkill in my opinion for just linting.